attributes/keycloak.rb

```diff
@@ -0,0 +1,22 @@
+# encoding: UTF-8
+#
+# Cookbook Name:: keycloak
+#
+# Copyright (C) 2017 Rodel Talampas
+# Updated cookbook of Janos Schwellach for keycloak
+# based on the wildfly cookbook from Brian Dwyer (https://github.com/bdwyertech/chef-wildfly)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+default['keycloak']['usercache']['enabled'] = false
```

Comment:
default should be true as keycloak has true as well

Reply:
noted